packages/Statie/packages/Generator/src/ObjectSorter.php

```diff
@@ -0,0 +1,23 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\Statie\Generator;
+
+use Symplify\Statie\Generator\Contract\ObjectSorterInterface;
+use Symplify\Statie\Renderable\File\AbstractFile;
+
+final class ObjectSorter implements ObjectSorterInterface
```

Comment:
I'd change the name to something like `FilenameObjectSorter` or `DescFilenameObjectSorter`.

It'll be more clear to know what's going on.

Reply:
That makes sense :+1: